components/upload/Upload.tsx

```diff
@@ -238,12 +238,13 @@ class Upload extends React.Component<UploadProps, UploadState> {
   };
 
   renderUploadList = (locale: UploadLocale) => {
-    const { showUploadList, listType, onPreview } = this.props;
+    const { showUploadList, listType, onPreview, previewFile } = this.props;
     const { showRemoveIcon, showPreviewIcon } = showUploadList as any;
     return (
       <UploadList
         listType={listType}
         items={this.state.fileList}
+        previewFile={previewFile}
```

Comment:
previewImage

Reply:
File can be video or other non-image type. `previewFile` may better than image.